### PR TITLE
manifest: Bump nrxlib revision + osif button init fix [KRKNWK- 13598]

### PR DESCRIPTION
**Pull request by sebastiandraus**
- Fixes zigbee osif button init function
- Updates nrxlib revision to include changes inthe zboss library.

### CODE REVIEW

**Review by tomchy — APPROVED** [review]
Please update release notes file too :slightly_smiling_face:

**Comment by sebastiandraus** [comment]
Added entry in release notes changelog

**Review by greg-fer — APPROVED** [review]

**Comment by sebastiandraus** [comment]
Fixed merge conflicts and compliance issue

**Comment by sebastiandraus** [comment]
Rebased at newest main branch

File: west.yml [hunk 1]
```diff
@@ -109,7 +109,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 5a88e0c14dbaf6233100c2b4827bd83545ae0da2
+      revision: pull/718/head
```
Comment (rlubos) [inline west.yml:112]: Please update SHA: `410ee7791eb26ff28607b4b1a7b24cac451f91f3`

**Review by rlubos — APPROVED** [review]

**Pull request merged**